tests fail

```
I: pybuild base:170: python2.7 ./runtests.py
............................F..........s.s.s.........
======================================================================
FAIL: test_post_location_list_EWKT (django_restframework_gis_tests.tests.TestRestFrameworkGis)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/django_restframework_gis_tests/tests.py", line 137, in test_post_location_list_EWKT
    expected_coords
AssertionError: Tuples differ: (6.381496, 53.384067) != (6.381495826183805, 53.3840669...

First differing element 0:
6.381496
6.38149582618

- (6.381496, 53.384067)
+ (6.381495826183805, 53.384066927384985)

----------------------------------------------------------------------
Ran 53 tests in 0.355s

FAILED (failures=1, skipped=3)
Creating test database for alias 'default'...
Destroying test database for alias 'default'...
```

--- a/tests/django_restframework_gis_tests/tests.py
+++ b/tests/django_restframework_gis_tests/tests.py
@@ -132,10 +132,8 @@ class TestRestFrameworkGis(TestCase):
         expected_coords = (6.381495826183805, 53.384066927384985)
         self.assertEqual(response.status_code, 201)
         self.assertEqual(Location.objects.count(), 1)
-        self.assertEquals(
-            Location.objects.get(name='EWKT input test').geometry.coords,
-            expected_coords
-        )
+        for l, e in zip(Location.objects.get(name='EWKT input test').geometry.coords, expected_coords):
+            self.assertAlmostEqual(l, e, places=5)
 
     def test_post_location_list_WKT_as_json(self):
         self.assertEqual(Location.objects.count(), 0)

